### PR TITLE
fix(meetings): sending ice candidate errors with add media success metric

### PR DESCRIPTION
**Pull request by marcin-bazyl**
Submitting info about candidate errors also with `js_sdk_add_media_success` metric event (we already send it with `js_sdk_add_media_failures` metric) so that we can see how often we have issues with DNS resolution of Linus hostname used for TLS

### CODE REVIEW

**Review by k-wasniowski — APPROVED** [review]

**Review by arun3528 — APPROVED** [review]
LGTM just change the title to mention plugin-meetings

File: packages/@webex/plugin-meetings/src/meeting/index.ts [hunk 1]
```diff
@@ -7022,6 +7022,7 @@ export default class Meeting extends StatelessWebexPlugin {
         await this.mediaProperties.getCurrentConnectionInfo();
       // @ts-ignore
       const reachabilityStats = await this.webex.meetings.reachability.getReachabilityMetrics();
+      const iceCandidateErrors = Object.fromEntries(this.iceCandidateErrors);
```
Comment (mkesavan13) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:7025]: Is there a possibility that this `iceCandidateErrors` be `undefined` in any case? 
Screenshot 2024-09-12 at 11 58 56 AM

**Review by mkesavan13 — APPROVED** [review]
One minor comment and rest looks good to me. Approved

File: packages/@webex/plugin-meetings/src/meeting/index.ts (hunk 1 above)
Comment (marcin-bazyl) [reply packages/@webex/plugin-meetings/src/meeting/index.ts:7025]: no, it's never undefined, it's initialised to an empty Map in constructor and is cleared when new connections are created, but never set to undefined

Comment (mkesavan13) [reply packages/@webex/plugin-meetings/src/meeting/index.ts:7025]: Great. Thanks for the clarification

**Pull request merged**